data/korath/korath ships.txt

```diff
@@ -1360,3 +1149,327 @@ ship "Model 512"
 	explode "large explosion" 35
 	"final explode" "final explosion large"
 	description "This is the final state in the evolution of a Kor Mereti war drone: a perfect ring, with incredibly powerful shields and turrets firing in every direction."
+
+
+ship "Korath Ravager"
+	sprite "ship/ravager"
+	thumbnail "thumbnail/ravager"
+	attributes
+		category "Heavy Warship"
+		"cost" 3760000
+		"shields" 27000
+		"hull" 25000
+		"required crew" 319
+		"bunks" 338
+		"mass" 1260
+		"drag" 12.3
+		# Ravager has massive surface area per volume plus heat sinks:
+		"heat dissipation" .8
+		"fuel capacity" 900
+		"cargo space" 110
+		"outfit space" 897
+		"weapon capacity" 415
+		"engine capacity" 232
+		"ramscoop" 2
+		weapon
+			"blast radius" 300
+			"shield damage" 4400
+			"hull damage" 2200
+			"hit force" 6600
+	outfits
+		"Korath Inferno"
+		"Korath Digger Turret" 6
+		"Korath Warder" 2
+		"Firestorm Turret"
+		"Firestorm Torpedo Rack"
+		"Firestorm Torpedo" 60
+		
+		"Systems Core (Large)"
+		"Fuel Processor"
+		"Plasma Core"
+		"Generator (Candle Class)"
+		"Triple Plasma Core"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Korath Repeater Rifle" 219
+		
+		"Steering (Comet Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Planetary Class)"
+		"Jump Drive"
+
+	turret -46 -85 "Korath Digger Turret"
+	turret 46 -85 "Korath Digger Turret"
+	turret -8 -45 "Korath Digger Turret"
+	turret 8 -45 "Korath Digger Turret"
+	turret -21 5 "Korath Digger Turret"
+	turret 21 5 "Korath Digger Turret"
+	turret -150 21.5 "Korath Warder"
+	turret 150 21.5 "Korath Warder"
+	turret -102 81.5 "Korath Inferno"
+	turret 102 81.5 "Firestorm Turret"
+	engine -27 73.5 0.9
+	engine 27 73.5 0.9
+	explode "tiny explosion" 120
+	explode "small explosion" 60
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description `The Ravager was the last crewed warship the Korath designed before they switched to drone ships. Neither side knows who originated the design; each kept capturing the others' ships and reverse-engineering improvements. This unending design competition bred a ship uncharacteristically efficient for the Korath. Unfortunately, its vast crew requirement leaves little space for passengers, cargo, or outfits. Ravagers can handle a phenominal arsenal and enough engines to out-maneuver the fastest drones, but not both at the same time.`
+
+
+ship "Korath Dredger"
+	sprite "ship/dredger"
+	thumbnail "thumbnail/dredger"
+	attributes
+		category "Heavy Freighter"
+		"cost" 29370000
+		"shields" 39000
+		"hull" 16000
+		"required crew" 191
+		"bunks" 683
+		"mass" 1353
+		"drag" 16
+		"heat dissipation" .55
+		"fuel capacity" 1000
+		"cargo space" 480
+		"outfit space" 796
+		"weapon capacity" 294
+		"engine capacity" 185
+		
+		# Capabilities of built-in nanite factory:
```

Comment:
I like these nanite factories, but - especially since they are a unit unto themselves - it would make more sense to add a small, medium, and large nanite factory as outfits instead. So long as they are sized appropriately, I don't see a problem with being able to add more of them (or possibly strip them off for certain builds). 

Given that the Korath have a history with nanites, having this sort of outfit would make a lot of sense. It also complements the system cores rather nicely. System Cores are energy centric control hubs focused on shields, control, and energy. Nanites are focused on hull and structure. 

That being said, the Korath have what is generally considered to be a very good fuel outfit in the form of the fuel processor. I'd suggest not including ramscoops in the nanite factories, and instead add an extra fuel processor.

Reply:
The inflexibility of the ships is part of the balancing. A Dredger has some built-in shield and armor repair, but that comes with heat and energy drain that won't ever go away. That improves the balance against the Bactrian. The world-ships have a similar problem. It may be nice to halve the Refabricator's repair to turn into a gunboat, but you can't.

Amazinite didn't want more systems cores, so I won't make a outfit versions to equip.

Reply:
I just asked Amazinite about that, and he clarified that he's OK with having a strong hull repair outfit; he's just not a fan of a "system core except hull focused" thing.